runtime/src/bank/tests.rs

```diff
@@ -12833,6 +12834,75 @@ fn test_rebuild_skipped_rewrites() {
     assert_eq!(snapshot_skipped_rewrites, actual_skipped_rewrites);
 }
 
+/// Test that getting accounts for BankHashDetails works with skipped rewrites
+#[test_case(true; "skip rewrites")]
+#[test_case(false; "do rewrites")]
+fn test_get_accounts_for_bank_hash_details(skip_rewrites: bool) {
```

Comment:
nit: Given how large this file already is, what do you think about moving this into `bank_hash_details.rs` ?

Reply:
> For me, If I see a method on Bank, I expect to see its tests in bank/tests.rs

Good point